Sources/Parley/Parley.swift

```diff
@@ -295,7 +295,7 @@ public final class Parley {
     }
     
     private func clearMessagesAndDataSources() {
-        messagesManager.clear()
+        messagesManager?.clear()
```

Comment:
When you call `Parley.reset()` the app will crash, because some vars are implicitly unwrapped optionals. By optional chaining the app will not crash anymore.